### PR TITLE
ForwardingService: Combine the `main` and `forward` methods

### PR DESCRIPTION
**Pull request by msgilligan**
(This is partly in preparation for a forthcoming change that will simplify the main method.)

### CODE REVIEW

File: integration-test/src/test/java/org/bitcoinj/examples/ForwardingServiceTest.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ void setupTest() {
     public void startViaStaticForwardAndImmediatelyInterrupt(@TempDir File tempDir) {
```
Comment (schildbach) [inline integration-test/src/test/java/org/bitcoinj/examples/ForwardingServiceTest.java:46]: The test name reads like it now needs to be changed, too?

**Review by schildbach — CHANGES_REQUESTED** [review]
Looks good!

File: integration-test/src/test/java/org/bitcoinj/examples/ForwardingServiceTest.java (hunk 1 above)
Comment (msgilligan) [reply integration-test/src/test/java/org/bitcoinj/examples/ForwardingServiceTest.java]: Fixed and force-pushed.

**Comment by schildbach** [comment]
Merged.

**Pull request closed without merging**